security/subsystem/src/main/resources/schema/jboss-as-security_2_0.xsd

```diff
@@ -44,6 +44,7 @@
          <xs:element name="security-management" type="securityManagementType" minOccurs="0" maxOccurs="1"/>
          <xs:element name="security-domains" type="securityDomainsType" minOccurs="0" maxOccurs="1"/>
          <xs:element name="vault" type="vaultType" minOccurs="0" maxOccurs="1"/>
+         <xs:element name="elytron-integration" type="elytronIntegrationType" minOccurs="0" maxOccurs="1"/>
```

Comment:
As we are planning to merge this in WildFly 11 maybe we need to bump the version to the next along and make the schema 3 ?

This way 2 is left should any WildFly 10 modifications be needed.